cpp/src/bitmask/bitmask_ops.h

```diff
@@ -3,11 +3,28 @@
 
 #include <cuda_runtime.h>
 
-
+/**---------------------------------------------------------------------------*
+ * @brief Sets all bits in input valid mask to 1
+ * 
+ * @param valid_out preallocated valid mask to set the values for
+ * @param out_null_count number of nulls (0 bits) in valid mask. Always set to 0
+ * @param num_values number of values in column for which the output mask was made
+ * @param stream cuda stream to run in
+ * @return gdf_error 
+ *---------------------------------------------------------------------------**/
 gdf_error all_bitmask_on(gdf_valid_type * valid_out, gdf_size_type & out_null_count, gdf_size_type num_values, cudaStream_t stream);
```

Comment:
I feel like this should take a preallocated column and just set the fields in it. That way you don't have to pass `out_null_count` and `num_values`.

Reply:
Agreed. Should I do it in this PR? I didn't touch this function, just added docs while passing by.